picard/ui/scripteditor.py

```diff
@@ -254,6 +298,82 @@ def confirmation_dialog(parent, message):
     return dialog.exec_() == QtWidgets.QMessageBox.Ok
 
 
+def synchronize_vertical_scrollbars(widgets):
+    """Synchronize position of vertical scrollbars and selections for listed widgets.
+
+    Args:
+        widgets (list): List of QListView widgets to synchronize
+    """
+    # Set highlight colors for selected list items
+    example_style = widgets[0].palette()
+    highlight_bg = example_style.color(QPalette.Active, QPalette.Highlight)
+    highlight_fg = example_style.color(QPalette.Active, QPalette.HighlightedText)
+    stylesheet = "QListView::item:selected { color: " + highlight_fg.name() + "; background-color: " + highlight_bg.name() + "; }"
+
+    def _sync_scrollbar_vert(widget, value):
+        widget.blockSignals(True)
+        widget.verticalScrollBar().setValue(value)
+        widget.blockSignals(False)
+
+    widgets = set(widgets)
+    for widget in widgets:
+        for other in widgets - {widget}:
+            widget.verticalScrollBar().valueChanged.connect(partial(_sync_scrollbar_vert, other))
+        widget.setStyleSheet(stylesheet)
+
+
+def populate_script_selection_combo_box(naming_scripts, selected_script_id, combo_box):
+    """Populate the specified script selection combo box and identify the selected script.
+
+    Args:
+        naming_scripts (list): List of available user-defined naming scripts in YAML format
+        selected_script_id (str): ID code for the currently selected script
+        combo_box (QComboBox): Combo box object to populate
+
+    Returns:
+        int: The index of the currently selected script
+    """
+    SCRIPT_TITLE_USER = _("User: %s")
```

Comment:
We should get rid of `SCRIPT_TITLE_USER` and `ScriptEditorDialog.SCRIPT_TITLE_USER` and just create a new method at module level:

```python
def script_title_user(title):
    return _("User: %s") % title
```

Then replace all occurences like:

```python
SCRIPT_TITLE_USER % script_item["title"]
```

with:

```python
script_title_user(script_item["title"])
```